public/index.js

```diff
@@ -0,0 +1 @@
+console.log('Hello World!');
```

Comment:
Just threw this in here so you can instantly tell the script was loaded properly.